client/dashboard/domains/domain-transfer/select-ips-tag.tsx

```diff
@@ -0,0 +1,166 @@
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	ExternalLink,
+} from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import { ipsTagListQuery, ipsTagMutation } from '../../app/queries/domain-transfer';
+import Notice from '../../components/notice';
+
+interface SelectIpsTagProps {
+	domain: string;
+	isDomainLocked: boolean;
+}
+
+export default function SelectIpsTag( { domain, isDomainLocked }: SelectIpsTagProps ) {
+	const { data: ipsTagList } = useSuspenseQuery( ipsTagListQuery() );
+	const saveIpsTagMutation = useMutation( ipsTagMutation( domain ) );
+	const [ ipsTag, setIpsTag ] = useState< string | null >( null );
+	const [ isDialogOpen, setIsDialogOpen ] = useState( false );
+	const [ saveStatus, setSaveStatus ] = useState< 'success' | 'error' | '' >( '' );
+
+	const suggestions = ipsTagList.map( ( item ) => item.tag );
+
+	if ( isDomainLocked ) {
+		return <Text>{ __( 'The IPS tag cannot be set while the domain is locked.' ) }</Text>;
+	}
+
+	const onConfirm = () => {
+		if ( ipsTag === null ) {
+			return;
+		}
+		saveIpsTagMutation.mutate( ipsTag, {
+			onSuccess: () => {
+				setSaveStatus( 'success' );
+			},
+			onError: () => {
+				setSaveStatus( 'error' );
+			},
+			onSettled: () => {
+				setIsDialogOpen( false );
+			},
+		} );
+	};
+
+	const renderDialog = () => {
+		const registrar = ipsTagList.find( ( item ) => item.tag === ipsTag );
+		return (
+			<Modal
+				title={ __( 'Transfer Confirmation' ) }
+				onRequestClose={ () => setIsDialogOpen( false ) }
+			>
+				<VStack spacing={ 6 }>
+					<Text>
+						{ createInterpolateElement(
+							__( 'Please verify you wish to set the registrar for <domain/> to the following:' ),
+							{
+								domain: <strong>{ domain }</strong>,
+							}
+						) }
+					</Text>
+					<Text>
+						<strong>{ registrar?.tag }</strong>
+						{ registrar?.registrarName && <> ({ registrar?.registrarName })</> }
+					</Text>
+					<Text>
+						{ __(
+							'After submitting this tag change, the domain will no longer be in our system. ' +
+								'You will need to contact the new registrar to complete the transfer and regain ' +
+								'control of the domain.'
+						) }
+					</Text>
+					<HStack justify="flex-end" spacing={ 2 }>
+						<Button
+							onClick={ () => setIsDialogOpen( false ) }
+							disabled={ saveIpsTagMutation.isPending }
+						>
+							{ __( 'Cancel' ) }
+						</Button>
+						<Button
+							variant="primary"
+							isBusy={ saveIpsTagMutation.isPending }
+							onClick={ onConfirm }
+							disabled={ saveIpsTagMutation.isPending }
+						>
+							{ __( 'Submit' ) }
+						</Button>
+					</HStack>
+				</VStack>
+			</Modal>
+		);
+	};
+
+	const renderIpsTagSelect = () => {
+		return (
+			<>
+				<FormTokenField
+					__next40pxDefaultSize
+					__nextHasNoMarginBottom
+					label="IPS tag"
```

Comment:
This should be translated too no?

Reply:
In this case I don't think it's necessary, it's very specific domains jargon, every registrar use it in different languages.